### PR TITLE
Fix EZP-24344: Implement FieldDefinition form mapper for eztext

### PR DESCRIPTION
**Pull request by glye**
Fieldsetting in place, no testing yet.

https://jira.ez.no/browse/EZP-24344

- [x] Refactor for changes done by @lolautruche

### CODE REVIEW

File: lib/FieldType/Mapper/TextBlockFormMapper.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+class TextBlockFormMapper implements FieldTypeFormMapperInterface
+{
+    public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
+    {
+        $fieldDefinitionForm
+            ->add(
+                'textRows', 'integer', [
+                'required' => false,
+                'property_path' => 'fieldSettings[textRows]',
+                'label' => 'field_definition.eztext.text_rows',
```
Comment (lolautruche) [inline lib/FieldType/Mapper/TextBlockFormMapper.php]: CS (missing indentation)

**Comment by glye** [comment]
Fixed indentation.

**Comment by lolautruche** [comment]
You need to rebase :-)

Implementation looks OK :+1: 

Don't wait for my refactoring to be merged, we'll refactor in one go.
As for tests, have a look at this: http://symfony.com/doc/current/cookbook/form/unit_testing.html

**Comment by glye** [comment]
Rebased & refactored, tests are todo.

**Comment by glye** [comment]
Added Twig whitespace control, to be consistent with existing field_types.html.twig content

File: bundle/Resources/views/ContentType/field_types.html.twig [hunk 2]
```diff
@@ -27,3 +27,11 @@
         {{- form_widget(form.defaultValue) -}}
     </div>
 {% endblock %}
+
+{% block eztext_field_definition_edit %}
+    <div class="eztext-text-rows">
```
Comment (lolautruche) [inline bundle/Resources/views/ContentType/field_types.html.twig]: I'd better use `eztext-settings text-rows` (2 classes)

**Comment by glye** [comment]
Ready for review, ping @lolautruche :)

**Comment by lolautruche** [comment]
Besides my small comment, +1

**Comment by glye** [comment]
Fixed CSS classes, thanks.

**Comment by pspanja** [comment]
+1

**Pull request merged**